scripts/main/init.js

```diff
@@ -151,6 +151,35 @@ $(document).ready(function() {
 
 	})
 
+	// Paste upload
+	.on('paste', function (e) {
+		if (e.originalEvent.clipboardData.items) {
+			if (!album.isUploadable()) {
+				return false;
+			}
+
+			// Close open overlays or views which are correlating with the upload
+			if (visible.photo()) lychee.goto(album.getID());
+			if (visible.contextMenu()) contextMenu.close();
+
+			if (e.originalEvent.clipboardData.items.length > 0) {
+				const items = e.originalEvent.clipboardData.items;
+				let imagesToUpload = [];
+
+				// Search clipboard items for an image
+				for (let i = 0; i < items.length; i++) {
+					if (items[i].type.indexOf('image') !== -1) {
```

Comment:
Lychee also supports video files, so I guess you might want to test for them as well?

Reply:
Sure I'll check it out!

Reply:
@kamil4 Okay so I can't manage to copy a video to clipboard on Windows 10. Is it possible on Linux?

Reply:
No idea. I'm not sure if I've ever copied an image file over clipboard, let alone a video... But I intend to try when I test your PR this weekend :smile: